src/server.c

```diff
@@ -3566,9 +3566,9 @@ void createSharedObjects(void) {
     shared.loadingerr = createObject(OBJ_STRING,sdsnew(
         "-LOADING Redis is loading the dataset in memory\r\n"));
     shared.slowevalerr = createObject(OBJ_STRING,sdsnew(
-        "-BUSY Redis is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
+        "-BUSY Redis is busy running a job. for script You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
     shared.slowscripterr = createObject(OBJ_STRING,sdsnew(
-        "-BUSY Redis is busy running a script. You can only call FUNCTION KILL or SHUTDOWN NOSAVE.\r\n"));
+        "-BUSY Redis is busy running a job. for script You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
```

Comment:
i think you should revert these changes, and add a 3rd error message that's specific to modules.